sdk/eventhub/azure-eventhub/tests/livetest/asynctests/test_reconnect_async.py

```diff
@@ -169,18 +169,8 @@ async def test_send_connection_idle_timeout_and_reconnect_async(
             await asyncio.sleep(11)
             ed = transform_outbound_single_message(ed, EventData, amqp_transport.to_outgoing_amqp_message)
             sender._unsent_events = [ed._message]
-            await sender._send_event_data()
-
-    retry = 0
```

Comment:
can we check that error behavior is correct for receive_message_batch?

Reply:
discussed offline